docs/js/features/odata/odata-v4-client.mdx

```diff
@@ -24,7 +24,7 @@ For more details on **how to execute requests** using a (pre-)generated OData cl
 
 #### Filter on One-To-Many Navigation Properties
 
-OData V4 introduces the [lambda operators](http://docs.oasis-open.org/odata/odata/v4.01/odata-v4.01-part2-url-conventions.html#sec_LambdaOperators) e.g., `any`/`all`, so that the root property of the one-to-many navigation properties can be filtered. Below is an example that demonstrates how to use the lambda operator [any](/cloud-sdk/api/1.28.0/modules/sap_cloud_sdk_core#any)).
+OData V4 introduces the [lambda operators](http://docs.oasis-open.org/odata/odata/v4.01/odata-v4.01-part2-url-conventions.html#sec_LambdaOperators) e.g., `any`/`all`, so that the root property of the one-to-many navigation properties can be filtered. Below is an example that demonstrates how to use the lambda operator [any](https://sap.github.io/cloud-sdk/api/1.28.0/modules/sap_cloud_sdk_core#any)).
```

Comment:
Hmm, are you sure hardcoding is the only option? I think the issue here might be, that you are using the base url as part of your url, so the previous version was pointing to https://sap.github.io/cloud-sdk/cloud-sdk/api/1.28.0/modules/sap_cloud_sdk_core#any. Can you try the previous version but just remove `/cloud-sdk` in the beginning?

Also, there is one closing paranthese too much.

Reply:
The current `document` branch will generate `http://localhost:3000/cloud-sdk/api/1.28.0/modules/sap_cloud_sdk_core#any` in my local machine, so I think it should work.
Unfortunately, the pipeline failed.
So I searched for the usage in the `documentation`, and the destination [page](http://localhost:3000/cloud-sdk/docs/js/features/connectivity/destination-js-sdk) is using the whole url.
I also used the `/api/1.28.1/modules/sap_cloud_sdk_core#any` and it generated `http://localhost:3000/cloud-sdk/api/1.28.0/modules/sap_cloud_sdk_core#any`, which is the same as the current version.
We have to try it by merging.